jetty-http/src/main/java/org/eclipse/jetty/http/GZIPContentDecoder.java

```diff
@@ -37,13 +37,16 @@
  */
 public class GZIPContentDecoder implements Destroyable
 {
+    // Unsigned Integer Max == 2^32
+    private static final long UINT_MAX = 0xffffffffL;
```

Comment:
Please use upper case for hex "digits": `0xFFFFFFFFL`.

Reply:
Done